src/util/common-mixin.js

```diff
@@ -15,198 +15,215 @@
  *    limitations under the License.
  */
 
-import * as Util from './util'
```

Comment:
The whole file seem to have changed and makes it very difficult to actually review the changes.
Can you fix this?

Reply:
Fixed